scripts/spell.pl

```diff
@@ -116,10 +118,21 @@ sub cmd_spell {
 	Irssi::active_win()->print("spell: " . check_line($inputline, $guesses), MSGLEVEL_CRAP );
 }
 
+sub sig_config {
+	$Lingua::Ispell::path=Irssi::settings_get_str($IRSSI{'name'}.'_ispell_path');
 
+	my $dic=Irssi::settings_get_str($IRSSI{'name'}.'_use_dictionary');
+	Lingua::Ispell::use_dictionary($dic) if ($dic ne "") ;
+}
+
+
+Irssi::settings_add_str('misc', $IRSSI{'name'} . '_use_dictionary', "");
+Irssi::settings_add_str('misc', $IRSSI{'name'} . '_ispell_path', "/usr/local/bin/ispell");
```

Comment:
Add a comment indicating that this is the default value of $Lingua::Ispell::path.